README.md

```diff
@@ -33,7 +33,7 @@ We currently support `Windows`, `MacOS`, and `Linux` operating systems. The mini
 
 Before installing the plugin, please make sure you have the following installed:
 
-- Python 3.10 or later
+- Python 3.10 
```

Comment:
Actually @yrkim98 is working on a change right now to allow 3.9

Reason: bundled installation (via the plugins menu in a running napari instance) requires 3.9 support.